src/app/core/services/profile/profile.service.ts

```diff
@@ -171,4 +173,24 @@ export class ProfileService {
       }
     }).catch(error => {})
   }
+
+  async prefillData(requestDetails: any,entityNames:any,formData:any,showAddOption:any=true) {
```

Comment:
why are we doing prefill in profile service

Reply:
we are prefilling profile data so i kept that function in profile service